torch/quantization/custom_module_class_mappings.py

```diff
@@ -0,0 +1,75 @@
+OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS = dict()
+
+def register_observed_custom_module_mapping(custom_module_class, observed_custom_module_class):
```

Comment:
maybe we can call `custom_module_class` to be `fp_custom_module_class` or `orig_custom_module_class`, to make it clear that this is a different custom class from observed and quantized?

can we also have a docblock somewhere which clearly explains the (fp) -> (obs) -> (quant) flow in a single paragraph, ideally with a toy example?

Reply:
sure.

for the toy example, I think it's better to do it in a tutorial or the top level quantization doc. I can have a docstring in this file, but not sure how discoverable that is.